packages/chart/src/plotly/Plot.ts

```diff
@@ -0,0 +1,7 @@
+// Import the react customizable bundle for building, only pull in the modules we need
+// This reduces the build size. Plotly has a lot of modules we don't need/use.
+// https://github.com/plotly/react-plotly.js#customizing-the-plotlyjs-bundle
+import createPlotlyComponent from 'react-plotly.js/factory.js';
+import Plotly from './Plotly';
+
+export default createPlotlyComponent(Plotly);
```

Comment:
Verified this works from docs/Docusaurus?

Reply:
Reverted to the old version. Something is weird w/ the docs and the ESM/CJS compilation. In addition we lazy load it in the docs so that adds another layer